src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/template_build_task_create.json

```diff
@@ -172,7 +179,8 @@
                 "branch": "[parameters('sourceControlBranch')]",
                 "imageName": "[parameters('imageName')]",
                 "isPushEnabled": "[parameters('isPushEnabled')]",
-                "dockerFilePath": "[parameters('dockerFilePath')]"
+                "dockerFilePath": "[parameters('dockerFilePath')]",
+                "buildArguments": "[parameters('buildArguments')]"
```

Comment:
how come secret build args aren't supported here?

Reply:
current RP model combines it with property `isSecret `on `BuildArgument`.

Reply:
I see, ty